packages/cmr-client/src/CMR.ts

```diff
@@ -228,9 +266,8 @@ export class CMR {
 
     if (params.token) {
       if (this.oauthProvider === 'launchpad') headers.Authorization = params.token;
-      else headers['Echo-Token'] = params.token;
+      else headers['Edl-Token'] = params.token;
```

Comment:
Have you tested this?  Based on the [CMR doc](https://cmr.earthdata.nasa.gov/ingest/site/docs/ingest/api.html), bearer token is used for EDL token